ti-lvgl-demo/suite/trixie/debian/rules

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --buildsystem=cmake --sourcedir=lv_port_linux
+
+override_dh_auto_configure:
+	dh_auto_configure -- -DCMAKE_C_FLAGS="-I/usr/include/libdrm"
```

Comment:
There is almost always a better way to help CMAKE find the right headers than hardcoding it into C_FLAGS. Either fix the source with a patch, or check for a better variable in the CMake build files (maybe `libdrm_INCLUDE_DIRS` or similar).